### ISSUE TITLE
Cache TensorRT modules

### ISSUE THREAD

**Issue by sjuxax** [occ-ai/obs-backgroundremoval#549]
**Is your feature request related to a problem? Please describe.**

Every time I open the OBS with this plugin installed, there's a 60 sec+ delay while the TensorRT modules are compiled.

**Describe the solution you'd like**

Most applications use a cache to prevent this from occurring on each startup. That would be great.

**Describe alternatives you've considered**

Waiting a long time each time I start OBS, which is annoying.

**Comment by royshil** [comment]
do you have any details on precompilation of tensorRT modules?

**Comment by sjuxax** [comment]
Both [ONNX Runtime](https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#execution-provider-options) and [Tensorflow](https://docs.nvidia.com/deeplearning/frameworks/tf-trt-user-guide/index.html#max-cached-engines) provide mechanisms for caching TensorRT engines.

I don't have a whole lot of other information, unfortunately, other than it's worked well on the onnxruntime applications I've used.

**Comment by krakowski** [comment]
I forked this repository and added CUDA as an option which loads instantly. I could also look into the TensorRT option mentioned by @sjuxax.

@royshil Should I open a pull request with my changes? 
  * https://github.com/occ-ai/obs-backgroundremoval/compare/main...krakowski:obs-backgroundremoval:refactor/linux?expand=1

I have also made a small change to the build script, allowing me to set the ONNX Runtime version using variables.
